src/main/java/org/jabref/gui/entryeditor/citationrelationtab/semanticscholar/SemanticScholarFetcher.java

```diff
@@ -86,10 +89,10 @@ public List<BibEntry> searchCiting(BibEntry entry) throws FetcherException {
 
     @Override
     public String getName() {
-        return "Semantic Scholar Citations Fetcher";
+        return FETCHER_NAME;
     }
 
     private String getApiKey() {
-        return importerPreferences.getApiKey(getName()).orElse(API_KEY);
+        return importerPreferences.getApiKey(getName()).orElse(apiKey);
```

Comment:
The fallback is wrong; the default key is now in the ImporterPreferences? Here, `""``  should be returned?